cypress/e2e/saucedemo/VerifyingDatacy.ts

```diff
@@ -0,0 +1,25 @@
+
+import { STANDART_USER, PASSWORD } from "../../Constants";
+describe('checkout test', () => {
+  beforeEach(() => {
+    cy.visit('https://www.saucedemo.com/');
+    cy.get('[data-test="username"]').type(STANDART_USER)
+    cy.get('[data-test="password"]').type(PASSWORD)
+    cy.get('#login-button').click()
+
+  })
+  it('Checkout test', () => {
+    // cy.type('Cypress.io{enter}');
+    cy.get('#add-to-cart-sauce-labs-bike-light').click();
+    cy.get('.shopping_cart_badge').click();
+    cy.get('#checkout').click();
+    cy.get('[data-test="firstName"]').type('Anton'); //https://docs.cypress.io/api/cypress-api/custom-commands
```

Comment:
no need comment here

Reply:
its for myself personally so that to remember from which source has been got